specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/examples/GetExtension.json

```diff
@@ -9,8 +9,8 @@
   "responses": {
     "200": {
       "body": {
-        "workspaceId": "id",
-        "primaryKey": "key"
+        "clusterMonitoringEnabled": true,
```

Comment:
This change in example file is in order to keep consistent with the change in swagger spec file.